vendor/bat-native-ledger/src/bat/ledger/internal/endpoint/gemini/gemini_utils.cc

```diff
@@ -0,0 +1,92 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "bat/ledger/internal/endpoint/gemini/gemini_utils.h"
+
+#include "base/base64.h"
+#include "base/strings/stringprintf.h"
+#include "bat/ledger/internal/gemini/gemini_util.h"
+#include "bat/ledger/internal/logging/logging.h"
+#include "bat/ledger/ledger.h"
+#include "net/http/http_status_code.h"
+
+namespace ledger {
+namespace endpoint {
+namespace gemini {
+
+std::string GetClientId() {
+  return ::ledger::gemini::GetClientId();
+}
+
+std::string GetClientSecret() {
+  return ::ledger::gemini::GetClientSecret();
+}
+
+std::vector<std::string> RequestAuthorization(const std::string& token) {
+  std::vector<std::string> headers;
+
+  if (!token.empty()) {
+    headers.push_back("Authorization: Bearer " + token);
+    return headers;
+  }
+
+  const std::string id = GetClientId();
+  const std::string secret = GetClientSecret();
+
+  std::string user;
+  base::Base64Encode(base::StringPrintf("%s:%s", id.c_str(), secret.c_str()),
+                     &user);
+
+  headers.push_back("Authorization: Basic " + user);
+  return headers;
+}
+
+std::string GetApiServerUrl(const std::string& path) {
+  DCHECK(!path.empty());
+
+  std::string url;
+  if (ledger::_environment == type::Environment::PRODUCTION) {
+    url = GEMINI_API_URL;
+  } else {
+    url = GEMINI_API_STAGING_URL;
```

Comment:
Is staging used for both `staging` and `development` environments?

Reply:
yes, we just use the staging URL for both staging and development